root/etc/cont-init.d/98-crowdsec

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/with-contenv bash
+
+CONFIG_PATH="/config/crowdsec/"
+LIB_PATH="/usr/local/lua/crowdsec/"
+DATA_PATH="/var/lib/crowdsec/lua/"
+
+echo "**** Configuring CrowdSec nginx Bouncer ****"
+
+# If API keys are missing, disable mod and exit
+if [[ -z $CROWDSEC_API_KEY ]] || [[ -z $CROWDSEC_LAPI_URL ]]; then
+    echo "**** Missing API key or CrowdSec LAPI URL, cannot configure bouncer ****"
+    sed -i '/#Include CrowdSec Bouncer/d' /config/nginx/nginx.conf
+    sed -i '/include \/config\/nginx\/crowdsec_nginx.conf;/d' /config/nginx/nginx.conf
+    exit 1
+fi
+
+apk add -U --upgrade --no-cache \
+    gettext \
+    lua5.1 \
+    lua5.1-cjson \
+    lua-resty-http \
+    lua-sec \
+    nginx-mod-http-lua
+
+# Download nginx bouncer
+CROWDSEC_VERSION=$(curl -sX GET "https://api.github.com/repos/crowdsecurity/cs-nginx-bouncer/releases/latest" | awk '/tag_name/{print $4;exit}' FS='[""]');
```

Comment:
I think having the option to enforce a url would be beneficial here

Reply:
For applying a fixed version you mean?

Reply:
yea

Reply:
Done.